stdlib/public/runtime/Errors.cpp

```diff
@@ -288,6 +296,43 @@ swift_deletedMethodError() {
                     "fatal error: call of deleted method\n");
 }
 
+static void *Swift_Framework_Token = NULL;
+static void *Swift_OSan_Impl;
+
+SWIFT_RUNTIME_EXPORT
+void
+swift_osan_unwrap(bool aural) {
+  if (Swift_Framework_Token == NULL) {
+    if (aural) {
+      Swift_Framework_Token = dlopen("/System/Library/Frameworks/AudioToolbox.framework/Versions/A/AudioToolbox", RTLD_FIRST);
```

Comment:
I'd love to bring such an important feature to Windows. This is too platform specific :'(